plugin-api/src/main/java/net/clgd/ccemux/api/emulation/EmulatedPalette.java

```diff
@@ -90,21 +78,38 @@ public void resetColours() {
 	 * Gets a color's current value
 	 */
 	@Override
+	@Nonnull
 	public double[] getColour(int colour) {
 		return delegate.getColour(colour);
 	}
 
 	/**
 	 * Adds a listener that will be invoked when a color is changed
 	 */
-	public void addListener(ColorChangeListener listener) {
+	public void addListener(@Nonnull ColorChangeListener listener) {
 		listeners.add(listener);
 	}
 
 	/**
 	 * Removes a listener
 	 */
-	public void removeListener(ColorChangeListener listener) {
+	public void removeListener(@Nonnull ColorChangeListener listener) {
 		listeners.remove(listener);
 	}
+
+	/**
+	 * Whether a color in this palette has been changed
+	 */
+	@SuppressWarnings("all")
+	public boolean isChanged() {
+		return this.changed;
+	}
+
+	/**
+	 * Whether a color in this palette has been changed
+	 */
+	@SuppressWarnings("all")
```

Comment:
Why is this needed? I assume we're doing something odd with CC interaction?

Reply:
It was generated by lombok and I missed it